src/components/Button.js

```diff
@@ -1,16 +1,25 @@
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { value } = props;
-  return <button className="button" type="button">{value}</button>;
+  const { value, bgColor, width } = props;
+
+  return (
+    <button className="button" type="button" style={{ backgroundColor: bgColor, width: `${width}%` }}>
+      {value}
+    </button>
+  );
 };
 
 Button.propTypes = {
   value: PropTypes.string,
+  bgColor: PropTypes.string,
+  width: PropTypes.number,
 };
 
 Button.defaultProps = {
   value: '',
+  bgColor: '#e0e0e0',
+  width: 25,
```

Comment:
- [ ] [REQUIRED] width is `boolean`, true means that button take 50% of space (used for 0 button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))

Reply:
'width' prop converted to boolean value